docs/src/modules/branding/BrandingPersona.tsx

```diff
@@ -83,12 +83,20 @@ export default function BrandingPersona(props: BrandingPersonaProps) {
           <IconButton
             component="a"
             href={`https://twitter.com/${twitter}`}
+            target="_blank"
+            rel="noopener"
```

Comment:
@oliviertassinari added target _blank here to open new tab when click. I like like UX than replace the page if you are okay with it. 😊